internal/testutil/reindent.go

```diff
@@ -39,8 +39,8 @@ func Reindent(in string) []byte {
 	return buf.Bytes()
 }
 
-// IndentLines indents each line in the provided string with the prefix.
-func IndentLines(text string, prefix string) string {
+// PrefixEachLine indents each line in the provided string with the prefix.
+func PrefixEachLine(text string, prefix string) string {
 	var result strings.Builder
 	for _, line := range strings.Split(text, "\n") {
 		newLine := prefix + line + "\n"
```

Comment:
Now that I am seeing the test, do you think this function should add a newline to the last one?
```
foo
```
becomes 
```
prefixfoo

```
and I am wondering if it should be
```
prefixfoo
```
instead

Reply:
Yeah, it shouldn't. I didn't care much seeing this was used in tests. Updated in https://github.com/letFunny/chisel/pull/1/commits/b94720b970b0ffdc5523fe35116831d876f320a9.